tests/cypress/integration/skeleton/skeletons_import_export.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Importing and exporting skeletons', () => {
+    const fileToUpload = 'mounted_file_share/svg/upload.svg';
+    const { downloadsFolder } = require('../../../cypress.json');
+
+    before(() => {
+        cy.task('createFolder', downloadsFolder);
+    });
+
+    beforeEach(() => {
+        cy.loginHeadless();
```

Comment:
Not sure we need to do login before each test

Reply:
Cypress intentionally clears cookies between tests to reset the state. Not sure how it works in other tests yet